codegenerator/cli/templates/static/codegen/src/ink/components/CustomHooks.res

```diff
@@ -38,7 +34,7 @@ module InitApi = {
     {
       envioVersion,
       envioApiToken,
-      ecosystem: ecosystem.contents,
+      ecosystem: (config.ecosystem :> ecosystem),
```

Comment:
I'm guessing the type system would shout if the `@as` field were different here? Any reason not to just use the same type here as the one in config?

Reply:
Yep